src/test/kotlin/hwr/oop/tnp/MonsterTest.kt

```diff
@@ -0,0 +1,44 @@
+package hwr.oop.tnp
+
+import io.kotest.core.spec.style.AnnotationSpec
+import org.assertj.core.api.Assertions.assertThat
+
+class MonsterTest : AnnotationSpec() {
+    private val stats = BattleStats(200, 20, 20, 20, 40, 40)
+    private val monstertype = Type("wasser")
```

Comment:
Please keep in it english and you need to change this anyway when you use a enum class
```suggestion
    private val monstertype = Type("Water")
```